### DISCUSSION TITLE
Dynamic Texture Generation Model Viewer

### DISCUSSION THREAD

**Discussion by Lemovision** [google/model-viewer#3972] [Show and tell]
Hi I would like to share this little project I made using model viewer

![ScreenNew](https://user-images.githubusercontent.com/61175245/204050355-247a7f78-96b0-42b1-a4e1-9f7d3667b008.png)

**https://www.elephantparade.com/digital-artbox**

Mostly worked on this last year actually, but they finally decided to release it on their website today
So the organization is Elephant Parade, who makes painted elephant statues with artists all around the world
They have exhibitions and sell replicas, partly contributing to conservation funds for elephants in Thailand

![rendertest](https://user-images.githubusercontent.com/61175245/204050375-f4382418-d63e-487f-b204-1b76c9abd6e7.png)

And to share this creative endeavor with their fans around the world, I suggested a digital painting experience
This relies on Unity with a painting plugin for the design part, to which I added plenty of tools and painting options
When painting is done, Unity exports the texture to the server, which creates a link to share with friends

**https://www.elephantparade.com/view/?e=5**

This sharing part of course relies on model-viewer, with fully dynamic texture loading from what was just painted
If anyone try this, make sure to use the save and share button to upload your designs
Have a good painting day!

**Comment by rawwerks** [comment]
this is really cool! way to go.

**Comment by elalish** [comment]
Thanks for sharing! [Tweeted](https://twitter.com/modelviewer/status/1598356770589536258).

**Comment by Lemovision** [comment]
Updated now with elephant animation for any generated designs!

https://www.elephantparade.com/view/?e=5 (click walk)

PS @elalish some minor issue
https://github.com/google/model-viewer/discussions/3985
